### ISSUE TITLE
match_phrase

### ISSUE THREAD

**Issue by sliontc** [danpaz/bodybuilder#13]
When will this query type be supported?

**Comment by danpaz** [comment]
Hey @sliontc, thanks for pointing that out! Looks like it wouldn't be too hard to add support for match_phrase. Based on [these docs](https://www.elastic.co/guide/en/elasticsearch/guide/current/phrase-matching.html) one can either use the match_phrase query clause, or modify the match query clause with a `type: "phrase"`. Either approach WFM. Want to make a PR for that?

**Comment by danpaz** [comment]
Resolved by #27. As of 1.9.0 the match query now accepts an additional optional argument to include several different options, including `type: 'phrase'`. For example:

```
var body = new Bodybuilder().query('match', 'message', 'this is a test', {type: 'phrase'})
```
